teachers_behavior.rb

```diff
@@ -0,0 +1,17 @@
+module Salary
```

Comment:
Since these are behaviors/properties of both Teacher and Apprentice Teacher, and those are both Faculty (or Staff, or Educators), it makes a bit more sense to make this a common superclass for the two kinds of teachers rather than a module.  Modules make more sense when the two entities that share behaviors are not the same *kind* of thing.